cmd/virt-handler/virt-handler.go

```diff
@@ -36,7 +36,6 @@ import (
 	kvtls "kubevirt.io/kubevirt/pkg/util/tls"
 	"kubevirt.io/kubevirt/pkg/virt-handler/seccomp"
 	"kubevirt.io/kubevirt/pkg/virt-handler/vsock"
-	"kubevirt.io/kubevirt/pkg/watchdog"
```

Comment:
We cannot remove it because it is still valid logic. The thing is we did have 2 watchdog mechanisms. One is purely functioning on files and the second one on sockets. In this PR we are just removing one of the implementations.